frontend/cypress/integration/featureFiles/istio_config.feature

```diff
@@ -78,14 +78,13 @@ Feature: Kiali Istio Config page
     Then the user can create a "Sidecar" Istio object
 
   @multi-cluster
-  @skip
+  @remote-istio-crds
   Scenario: See all Istio Config objects in the bookinfo namespace in the multi-cluster environment.
+    And there are Istio config objects created in both clusters
     Then the "Cluster" column "appears"
-    And user sees all the Istio Config objects from both clusters in the bookinfo namespace
-    And user sees Name information for Istio objects
-    And user sees Namespace information for Istio objects
-    And user sees Type information for Istio objects
-    And user sees Configuration information for Istio objects
+    And user sees the Istio Config objects from both clusters in the bookinfo namespace
+    And user sees "Name" information for Istio objects from both clusters
+    And user sees "Configuration" information for Istio objects from both clusters
```

Comment:
This test will fail unless it's a multi-primary deployment since by definition there are no CRDs present in a primary-remote deployment. Personally I'd prefer we use a `@multi-primary` and/or `@primary-remote` rather than `@remote-istio-crds`.

Let me know what you think but I see there being three different tags to use here and two different ways to use them:

Option 1: Everything tagged with multi-cluster. Need to filter out primary-remote and multi-primary in test execution.
`@multi-cluster`: Every multi-cluster test should be tagged with this.
`@primary-remote`: These only run with a primary-remote deployment.
`@multi-primary`: These only run with a multi-primary deployment.

Option 2: Only tests that can run on _both_ multi-primary _and_ primary-remote are tagged with `@multi-cluster`. Tests tagged with `@multi-primary` and `@primary-remote` are not tagged with `@multi-cluster`.
`@multi-cluster`: Only tests that can run on _both_ multi-primary _and_ primary-remote should be tagged with this.
`@primary-remote`: These only run with a primary-remote deployment.
`@multi-primary`: These only run with a multi-primary deployment.